manifests/proxydhcp.pp

```diff
@@ -53,14 +60,18 @@
 
   if $foreman_proxy::dhcp_manage_acls {
 
-    package {'acl':
-      ensure => 'installed',
+    if $foreman_proxy::dhcp_leases =~ /(.*)\/(.*\.leases)/ {
```

Comment:
Why is this needed? I'm also wondering if this will survive. I think that dhcpd rewrites its lease file and creates a new one. Not sure if the ACLs would persist service restarts.

Reply:
that's why we take only the folder. I did not found a function that would take a file path and got directory name from that.

Reply:
Ah, so why not use [dirname() from stdlib](https://forge.puppet.com/puppetlabs/stdlib/reference#dirname)?

Reply:
Because you don't know about it's existance and you're unable to google it (and I've tried!).
me-- :)